src/DataFrames.jl

```diff
@@ -92,6 +93,7 @@ include("dataframerow/utils.jl")
 include("abstractdataframe/iteration.jl")
 include("abstractdataframe/join.jl")
 include("abstractdataframe/reshape.jl")
+include("dataframe/reshape.jl")
```

Comment:
Why put this in a file called `reshape`? Also, does it actually need a separate file?